examples/FirebaseNeoPixel_ESP8266/FirebaseNeoPixel_ESP8266.ino

```diff
@@ -0,0 +1,65 @@
+//
```

Comment:
That seems to be discussing when you do git rm/add I think if you do "git mv" than it will explicitly keep the link.